draft-ietf-rtcweb-jsep.xml

```diff
@@ -621,12 +621,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             attribute, it MUST first check to see if the original resolution
             meets the criteria specified in the attribute, and transmit it
             untouched if so. If the original resolution is too large for the
-            attribute criteria, the sender SHOULD apply downscaling to the
+            attribute criteria, the sender downscales the
```

Comment:
This seems an unnecessary change. If you think the policy is wrong, let's debate that.